app-shared/src/androidJvmMain/kotlin/io/github/droidkaigi/confsched/KaigiAppUi.androidJvm.kt

```diff
@@ -171,7 +172,20 @@ actual fun KaigiAppUi() {
             },
             modifier = Modifier
                 .fillMaxSize()
-                .hazeSource(hazeState),
+                .hazeSource(hazeState)
+                .bindMouseBackForward(
```

Comment:
https://github.com/DroidKaigi/conference-app-2025/pull/554#discussion_r2325326310
bindMouseBackForward is defined in the expect/actual method and does not affect other platforms such as Android or iOS. There are no issues as expressed in the above comment.
It should not hinder the review process.